File: blinkeye-framewk-comun-dominio/src/main/java/net/royal/spring/contabilidad/dominio/filtro/FiltroComunAcSucursal.java
package net.royal.spring.contabilidad.dominio.filtro;

import net.royal.spring.framework.modelo.generico.DominioPaginacion;

public class FiltroComunAcSucursal {

	private String sucursal;
	private String descripcionlocal;
	private String estado;
	private String companiasocio;
	private String tipoexportar;
	private DominioPaginacion paginacion = new DominioPaginacion();

	
	public String getTipoexportar() {
		return tipoexportar;
	}

	public void setTipoexportar(String tipoexportar) {
		this.tipoexportar = tipoexportar;
	}

	public String getSucursal() {
		return sucursal;
	}

	public void setSucursal(String sucursal) {
		this.sucursal = sucursal;
	}

	public String getDescripcionlocal() {
		return descripcionlocal;
	}

	public void setDescripcionlocal(String descripcionlocal) {
		this.descripcionlocal = descripcionlocal;
	}

	public String getEstado() {
		return estado;
	}

	public void setEstado(String estado) {
		this.estado = estado;
	}

	public String getCompaniasocio() {
		return companiasocio;
	}

	public void setCompaniasocio(String companiasocio) {
		this.companiasocio = companiasocio;
	}

	public DominioPaginacion getPaginacion() {
		return paginacion;
	}

	public void setPaginacion(DominioPaginacion paginacion) {
		this.paginacion = paginacion;
	}
	
	
}

File: web-deteccion/src/app/royal/modulos/framework-comun/contabilidad/dominio/filtro/FiltroComunAcSucursal.ts
import { DominioPaginacion } from './../../../../../framework/modelo/generico/DominioPaginacion';
export class FiltroComunAcSucursal {
    constructor() {
        this.paginacion = new DominioPaginacion();
    }
    paginacion: DominioPaginacion;

    sucursal: string;
    descripcionlocal: string;
    estado: string;
    companiasocio: string;
    tipoexportar: string;

 
    sucursalgrupo : string;
    descripcioningles : string;
    ultimousuario : string;
    ultimafechamodif : Date;
    cuentacontabledefecto : string;
    almacencodigo : string;
    racionperfil : string;
    departamento : string;
    provincia : string;
    codigopostal : string;
    pais : string;
    direccion : string;
    responsable : number;
   
}